packages/public/server/docker/php/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN sed -ie 's/# en_GB.UTF-8 UTF-8/en_GB.UTF-8 UTF-8/g' /etc/locale.gen
 RUN sed -ie 's/# de_DE.UTF-8 UTF-8/de_DE.UTF-8 UTF-8/g' /etc/locale.gen
 RUN sed -ie 's/# hi_IN UTF-8/hi_IN UTF-8/g' /etc/locale.gen
 RUN sed -ie 's/# ta_IN UTF-8/ta_IN UTF-8/g' /etc/locale.gen
+RUN sed -ie 's/# fr_FR UTF-8/fr_FR UTF-8/g' /etc/locale.gen
```

Comment:
I think this line is wrong: there's a difference between languages that also have an ISO-8859 representation (see the line for german or spanish for that) and those that need UTF-8 (like tamil or hindi)

Reply:
Good point @Knorrke Just updated it in the last commit and will put this in documentation.